library/src/main/java/com/novoda/downloadmanager/lib/DownloadInfo.java

```diff
@@ -47,7 +47,8 @@ public DownloadInfo newDownloadInfo(
                 SystemFacade systemFacade,
                 StorageManager storageManager,
                 DownloadNotifier notifier,
-                DownloadClientReadyChecker downloadClientReadyChecker) {
+                DownloadClientReadyChecker downloadClientReadyChecker,
+                Downloads downloads) {
```

Comment:
@blundell https://github.com/novoda/download-manager/pull/73#discussion_r34058092 and https://github.com/novoda/download-manager/pull/73#discussion_r34059506

@amlcurran Happy to do that in this PR but only after https://github.com/novoda/download-manager/pull/63 is merged, I don't want to have a hell of a conflicts to solve